azureDatabaseUtils/build.gradle

```diff
@@ -0,0 +1,54 @@
+// Build Script Classpath
+buildscript {
+    ext {
+        springBootVersion = '2.7.10'
```

Comment:
Can this use 3.x? We are having troubles upgrading to that (https://broadworkbench.atlassian.net/browse/WOR-649), but it's certainly our goal so if this new package can run with 3.x it would make sense to start it out that way.